### PR TITLE
fibjs 0.6.1

### PR DESCRIPTION
**Pull request by dunn**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/fibjs.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Fibjs < Formula
   desc "JavaScript on Fiber"
   homepage "http://fibjs.org/"
-  url "https://github.com/fibjs/fibjs.git",
-      :tag => "v0.5.0",
-      :revision => "92a0d14ee241fffd29603963cafb1c1bc9a0dcf8"
+  url "https://github.com/fibjs/fibjs/releases/download/v0.6.1/fullsrc.zip"
+  version "0.6.1"
```
Comment (ilovezfs) [inline Formula/fibjs.rb]: this is going to make it harder to maintain with livecheck

Comment (dunn) [inline Formula/fibjs.rb]: Does that outweigh the advantages of a proper source archive that doesn't require all the submodules?

Comment (ilovezfs) [inline Formula/fibjs.rb]: Yes. If it ain't broke don't fix it.

Comment (dunn) [inline Formula/fibjs.rb]: OK ¯\\\_(ツ)\_\/¯

Comment (ilovezfs) [inline Formula/fibjs.rb]: @dunn I'd be fine with it if we can fix the `brew` detect-version-from-url thing to be able to understand `https://github.com/fibjs/fibjs/releases/download/v0.6.1/fullsrc.zip`, which doesn't sound impossible.

Comment (ilovezfs) [inline Formula/fibjs.rb]: Looks like the same issue is affecting these: archi-steam-farm, clojurescript, closure-stylesheets, drake, easy-git, eprover, metabase, musepack, nsq, nuget, and smlnj.

Comment (dunn) [inline Formula/fibjs.rb]: I hadn't updated the PR since I'm at work; but either approach is fine.

**Review by ilovezfs — APPROVED** [review]
LGTM since the version parsing will be handled by https://github.com/Homebrew/brew/pull/2827.

**Comment by ilovezfs** [comment]
:bowing_man: :rocket: :tada:

**Pull request closed without merging**